.gitlab/hpsf-gitlab-ci.yml

```
@@ -8,6 +8,7 @@ NVIDIA-GH200:
       -DCMAKE_CXX_COMPILER=`pwd`/bin/nvcc_wrapper
       -DCMAKE_CXX_FLAGS="-Werror=all-warnings -Werror"
       -DKokkos_ARCH_HOPPER90=ON
+      -DKokkos_ARCH_NATIVE=ON
```

```suggestion
      -DKokkos_ARCH_NATIVE=ON
      -DKokkos_ARCH_HOPPER90=ON
```
Nitpick: I'd slightly prefer collocating enabling the device backend with the device architecture

---

Sure.